superset-frontend/package-lock.json

```diff
@@ -77984,7 +77984,7 @@
         "@mapbox/geojson-extent": "^1.0.1",
         "@math.gl/web-mercator": "^3.2.2",
         "@types/d3-array": "^2.0.0",
-        "@types/mapbox__geojson-extent": "*",
+        "@types/mapbox__geojson-extent": "^1.0.0",
```

Comment:
Please make a separate PR for this package version update since it's not related to the commit